gtsfm/configs/deep_front_end.yaml

```diff
@@ -13,14 +13,18 @@ SceneOptimizer:
 
   two_view_estimator:
     _target_: gtsfm.two_view_estimator.TwoViewEstimator
-    corr_metric_dist_threshold: 3 # in px
+    eval_threshold_px: 3 # in px
+    estimation_threshold_px: 4 # for homography estimator
+    min_num_inliers_acceptance: 15
```

Comment:
`min_num_inliers_for_success`?

Reply:
I find num_ kinda redundant when theres a min_. min_inliers_acceptance is shorter with no meaning lost.

Reply:
i'm cool with that, i'll update accordingly

Reply:
On second though, I actually think `min_num_inliers_acceptance` is perhaps better, since otherwise `min_inliers_acceptance` might make people think these are the actual inliers. Inded this is consistent with COLMAP: https://github.com/colmap/colmap/blob/dev/src/estimators/essential_matrix.h#L60
```
  // The minimum number of samples needed to estimate a model.
  static const int kMinNumSamples = 5;
```

Reply:
I have one more followup comment. Why not use `min_num_verified_corr`? I prefer to call the output of verification as verified correspondences instead of inlier correspondence. **Inlier** should be reserved for things which are correct according to the ground truth.